docs/guides/software-lifecycle/continuous-testing/README.md

```diff
@@ -1,3 +1,647 @@
+6. **Triggering the Job:** You can configure the job to be triggered on each push to your repository by using Jenkins webhooks or polling SCM, depending on your preference and setup.
+
+
+#### 3.2 System Test Automation
+
+To aid in the automation of system tests, we suggest doing the following:
+
+1. Have a schedule for running system tests (e.g. nightly, weekly)
+2. Ensure software is built and published to repositories:
+   - Stand-alone components of your software should be independently released, built, or packaged to be published on public repositories
+   - (Optional) a final, single build of an integrated software consisting of multiple components is built, packaged, and published to a public repository
+3. Pull built or packaged artifacts from repositories and deploy software release (components or single package) to a virtualized environment
+4. Test the deployed release against a set of specified system tests
+
+The diagram below illustrates this concept:
+
+```mermaid
+graph TD
+    subgraph SoftwareDevelopment[Software Components]
+        Code[Code Development]
+        UnitTests{Unit Tests Success?}
+        BuildComponents[Build Stand-alone Components]
+        PublishArtifacts[(Publish Artifacts to Repositories)]
+    end
+    subgraph IntegratedSoftware[Integrated Software]
+        SingleIntegratedBuild{Single Integrated Build?}
+        BuildIntegratedSoftware[Build Integrated Software]
+        PublishIntegratedArtifact[(Publish Integrated Artifact to Repository)]
+    end
+
+    subgraph TestDeployment[Testing & Deployment, e.g. nightly / weekly]
+        PullArtifacts[Pull Artifacts]
+        DeployRelease[Deploy Software Release]
+        IntegrationTests[Test Against System Tests]
+    end
+
+    Code --> UnitTests
+    UnitTests -->|Yes| BuildComponents
+    UnitTests -->|No| Code
+    BuildComponents --> PublishArtifacts
+    SingleIntegratedBuild -->|Yes| BuildIntegratedSoftware
+    PublishArtifacts --> BuildIntegratedSoftware
+    BuildIntegratedSoftware --> PublishIntegratedArtifact
+    PullArtifacts --> DeployRelease
+    DeployRelease --> IntegrationTests
+    PublishArtifacts --> TestDeployment
+    PublishIntegratedArtifact --> TestDeployment
+```
+
+##### Types of System Tests
+
+You should outline the types of system tests you plan to implement in your `TESTING.md` file. We suggest the following types of tests to include:
+- Testing for integration
+  - Interaction between software components
+  - Interaction with external services, provided files, exchange of messages, etc.
+- Testing for security
+  - See existing [SLIM security best practices](/slim/docs/category/security) as part of your software development workflow.
+- Testing for performance and load
+- Testing user interfaces for gaps and compliance against policies
+
+##### Integration Tests Automation
+
+We recommend the following steps for integration tests automation:
```

Comment:
This is good @yunks128 but I'd suggest enhancing it to be more directly tied to the above diagram, which basically implies CI. I may even color code that diagram to the CT vs CI parts.